<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientDlg</class>
 <widget class="QDialog" name="PatientDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_commentaire">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Commentaire:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_commentaire">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>395</y>
     <width>451</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>251</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_priorite">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_tel">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Tel:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_cp">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>CP:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_date">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_ville">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Ville:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_nom">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Nom:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_nom">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_ville">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_cp">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_prenom">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_duree">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Durée:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_prenom">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Prénom:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_adresse">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_duree">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_priorite">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Priorité:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_adresse">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Adresse:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_tel">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDateEdit" name="dateEdit_debut">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>lineEdit_priorite</zorder>
   <zorder>lineEdit_duree</zorder>
   <zorder>lineEdit_adresse</zorder>
   <zorder>lineEdit_cp</zorder>
   <zorder>lineEdit_ville</zorder>
   <zorder>lineEdit_tel</zorder>
   <zorder>lineEdit_prenom</zorder>
   <zorder>lineEdit_nom</zorder>
   <zorder>label_prenom</zorder>
   <zorder>label_nom</zorder>
   <zorder>label_tel</zorder>
   <zorder>label_adresse</zorder>
   <zorder>label_cp</zorder>
   <zorder>label_ville</zorder>
   <zorder>label_date</zorder>
   <zorder>label_duree</zorder>
   <zorder>label_priorite</zorder>
   <zorder>dateEdit_debut</zorder>
  </widget>
  <widget class="QTableWidget" name="tableWidget_ress">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>191</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>191</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_submit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
       </font>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cencel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>textEdit_commentaire</zorder>
  <zorder>label_commentaire</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>tableWidget_ress</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEdit_nom</tabstop>
  <tabstop>lineEdit_prenom</tabstop>
  <tabstop>lineEdit_tel</tabstop>
  <tabstop>lineEdit_ville</tabstop>
  <tabstop>lineEdit_cp</tabstop>
  <tabstop>lineEdit_adresse</tabstop>
  <tabstop>dateEdit_debut</tabstop>
  <tabstop>lineEdit_duree</tabstop>
  <tabstop>lineEdit_priorite</tabstop>
  <tabstop>textEdit_commentaire</tabstop>
  <tabstop>tableWidget_ress</tabstop>
  <tabstop>pushButton_submit</tabstop>
  <tabstop>pushButton_cencel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
